### DISCUSSION TITLE
Error when trying to import SLEAP in Google Colaboratory

### DISCUSSION THREAD

**Discussion by galaxiahfast** [talmolab/sleap#2019] [Help!]
I am trying to follow the command guide to install SLEAP, but I am encountering several errors that I shouldn't have.
```python
# Desinstalar los paquetes 'opencv-python' y 'opencv-contrib-python' si están instalados
!pip uninstall -qqq -y opencv-python opencv-contrib-python

# Instalar el paquete 'sleap' desde PyPI, asegurando que sea la versión 1.3.3 o superior
!pip install -qqq "sleap[pypi]>=1.3.3"
``` 
![Captura de pantalla 2024-11-06 213101](https://github.com/user-attachments/assets/a6cd415a-3277-48ed-9de1-24e766eb8865)

**Comment by galaxiahfast** [comment]
I restart the system and an error appears in the code.
![image](https://github.com/user-attachments/assets/451db771-48aa-496b-b01b-e813f3668acc)

**Comment by eberrigan** [comment]
Hi @JULIANEMI,

Can you try this as the first step `!apt update && apt install cuda-11-8`.

Please see issue #1644.

Thanks,

Elizabeth

**Reply by galaxiahfast** [reply]
I only ran the three commands, and still, I get an error. I managed to train the model and everything, but I'm not sure if it's due to the NumPy package warning, which is preventing it from outputting the mAP and mAR metrics at the end.

`!apt update && apt install cuda-11-8
!pip uninstall -qqq -y opencv-python opencv-contrib-python
!pip install -qqq "sleap[pypi]>=1.3.3"
`
![image](https://github.com/user-attachments/assets/0af3b5e0-78b7-4041-83a3-8188d59711f4)

It's strange because I've trained before and got results. Could it be due to a recent update?

![Captura de pantalla 2024-10-06 210226](https://github.com/user-attachments/assets/710b3f25-0c9d-402a-8aec-3422090219d8)
